docs/shiki.config.ts

```diff
@@ -0,0 +1,4 @@
+export const shikiTheme = {
+  light: 'github-light-default',
+  dark: 'material-theme-ocean',
+};
```

Comment:
Next.js was complaining about exporting a `config` from the project root, seems like this used to be some pattern that is now deprecated

For context, this file will go completely when I'm done with code blocks updates